examples/replica/main.tf

```diff
@@ -0,0 +1,39 @@
+##############################################################################
+# Resource Group
+##############################################################################
+
+module "resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.resource_group == null ? "${var.prefix}-resource-group" : null
+  existing_resource_group_name = var.resource_group
+}
+
+##############################################################################
+# ICD postgresql database
+##############################################################################
+
+module "postgresql_db" {
+  source            = "../.."
+  resource_group_id = module.resource_group.resource_group_id
+  name              = "${var.prefix}-primary"
+  region            = var.region
+  resource_tags     = var.resource_tags
+}
+
+##############################################################################
+# ICD postgresql read-only-replica
+##############################################################################
+
+module "replicate_postgresql_db" {
```

Comment:
read_only_replica_postgresql_db may make more sense here as a name.

Reply:
Renamed the module as suggested.